File: algo/warmup/cpp/plusMinus.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
#include <iomanip>
using namespace std;

int main()
{
    setprecision(6);
    int size, temp;
    float Neg = 0.0, Pos = 0.0, Zer = 0.0;
    cin >> size;
    //vector<int> arr(size);
    for (int i = 0; i < size; i++)
    {
        cin >> temp;
        if (temp < 0)
            Neg++;
        else if (temp > 0)
            Pos++;
        else
            Zer++;
    }
    cout << Pos / size << endl;
    cout << Neg / size << endl;
    cout << Zer / size << endl;
    return 0;
}

File: algo/warmup/js/plusMinus.js
function plusMinus(arr) {
    let pos = 0;
    let neg = 0;
    let zero = 0;
    const len = arr.length;
    arr.map((elem) => {
        if (elem > 0) pos++;
        else if (elem < 0) neg++;
        else zero++;
    });
    console.log(pos / len);
    console.log(neg / len);
    console.log(zero / len);
}